tests/common/snappi/common_helpers.py

```diff
@@ -523,18 +581,27 @@ def config_ingress_lossless_buffer_alpha(host_ans, alpha_log2):
     return True
 
 
-def get_pfcwd_config_attr(host_ans, config_scope, attr):
+def get_pfcwd_config_attr(host_ans, config_scope, attr, asic_value='None'):
```

Comment:
asic_value=None

Reply:
its actually 'None' because get_multidut_snappi_ports appends asic_value to 'None'